Catappult/appcoins-wallet-android
# [feature/APPC-2456] VIP badge for whales

**What does this PR do?**

   Creates a VIP badge for users with level 9 or level 10

**Database changed?**

No

**Where should the reviewer start?**

- TransactionsActivity.java
- TransactionsViewModel.java
- layout_vip_badge.xml

**How should this be manually tested?**

  Open the app with a vip level wallet and check the badge and the link navigation. 

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-2456

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)

**Code Review Checklist**
